ios/Handlers/LoginHandler.swift

```diff
@@ -1,7 +1,7 @@
 protocol BridgeToLoginHandlerProtocol: AnyObject {
     func authenticateUser(_ profile: ONGUserProfile, authenticator: ONGAuthenticator?, completion: @escaping (ONGUserProfile?, Error?) -> Void)
     func setAuthPinChallenge(_ challenge: ONGPinChallenge?)
-    func handlePinAction(_ pin: String?, action: PinAction)
+    func handlePinAction(_ pin: String?, action: PinAction) -> Bool
```

Comment:
This is not desired way to handle errors in Swift. Better approach is to throw an error (more flexibility with describing what was the reason is extra advantage) and use `do-catch` (with `try` inside) construction when calling such function. Doable?

Reply:
Yes, we actually did the same on android.